azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh

```diff
@@ -67,13 +72,16 @@ test_step "Check ip address immediately after the move"
 ssh_node1 'ip a show eth0'
 ssh_node2 'ip a show eth0' | grep ${MY_FIP} || test_die "VirtualIP ${MY_FIP} should be on ${MYNAME}-vm-02"
 
+# use curl to probe the webserver using the frontend IP
+# until the reply coe from the VM-02
 wait_for_takeover "${MYNAME}-vm-02"
 
 # Slow down, take a break, then check again, nothing should be changed.
 test_step "Check again later"
 sleep 30
 test_web "${MYNAME}-vm-02"
 
+# Repeate the same but this time from VM-02 to VM-01
```

Comment:
s/Repeate/Repeat/

@coderabbitai Take this you looser AI. I have found a typo you haven't!

Reply:
@coderabbitai Stop pretending to be nice. You cannot fool me, AI. I know what are you planning!